scripts/deploy.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+export RPCD_LOGSTASH=${RPCD_LOGSTASH:-"FALSE"}
+export RPCD_AIO=${RPCD_AIO:-"FALSE"}
+
+# Enable logstash if wanted
+if [ "${RPCD_AIO}" != "yes" ]; then
+  if [ "${RPCD_LOGSTASH}" == "yes" ]; then
+    mv /etc/openstack_deploy/user_logging_variables.yml.example /etc/openstack_deploy/user_logging_variables.yml || exit 1
```

Comment:
I agree, along these lines I want to import osad's script-libraries so we can use that with the ansible runs as well.